go/vt/wrangler/switcher_interface.go

```diff
@@ -28,6 +28,7 @@ import (
 type iswitcher interface {
 	lockKeyspace(ctx context.Context, keyspace, action string) (context.Context, func(*error), error)
 	cancelMigration(ctx context.Context, sm *workflow.StreamMigrator)
+	legacyStopStreams(ctx context.Context, sm *workflow.StreamMigrator) ([]string, error)
```

Comment:
A comment here about why we have both `legacyStopStreams` and `stopStreams` and why we needed the new function will be useful.